subsys/usb/usb_device.c

```diff
@@ -863,14 +941,21 @@ static bool usb_handle_std_endpoint_req(struct usb_setup_packet *setup,
 		break;
 
 	case REQ_CLEAR_FEATURE:
-		if (setup->wValue == FEA_ENDPOINT_HALT) {
-			/* clear HALT by unstalling */
-			LOG_INF("... EP clear halt %x", ep);
-			usb_dc_ep_clear_stall(ep);
-			if (usb_dev.status_callback) {
-				usb_dev.status_callback(USB_DC_CLEAR_HALT, &ep);
+		if (sys_le16_to_cpu(setup->wValue) == FEA_ENDPOINT_HALT) {
```

Comment:
sys_le16_to_cpu() is not necessary here

Reply:
It was there. I didn't change that.

Reply:
Ahh sorry nvm... I had to add this by mistake when rebasing from older branches. I will remove that